.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -0,0 +1,161 @@
+name: Release - Publish draft
+
+on:
+  push:
+    tags:
+      # Catches v1.2.3 and v1.2.3-rc1
+      - v[0-9]+.[0-9]+.[0-9]+*
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Current release/rc version
+
+jobs:
+  get-rust-versions:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-unified:bullseye-1.75.0-2024-01-22-v20240109
```

Comment:
Do you really want to depend on an actual CI image to do that? What about the image version upgrade policy here?

Reply:
https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-unified/Dockerfile#L4
For example, toolchain versions could be extracted from this `Dockerfile` part. In case if you want to continue exact version pinning in the workflow definition, it still needs to be known how the CI image version bumping would be handled here (possible options could be "with the image update in the rest of CI" / "after updating the image in CI", etc.)

Reply:
Not really, but haven't found a better way. I track the versions of the CI image as it used in our cleanroom scripts and upgrade it when the new version is up. But it is a manual process as well

Reply:
Yeah, we'll definitely come up with something more automated here after we'll migrate from GitLab to GHA.

Reply:
You could tag the the `paritytech/ci-unified` with a `CI-latest` tag and then only use that tag for CI jobs.  
Has the downside that old jobs possibly are not re-runnable, for example when we want to do a patch release of an older version.

Reply:
@rcny Vova, I changed the way to get the rust version as you suggested. Now it is parsed from the `Dockerfile`

Reply:
Nice, thanks!